tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -32,6 +32,8 @@ class Wifi(Base):
     _manage_certs_screen_locator = (By.ID, 'wifi-manageCertificates')
     _import_certs_locator = (By.CSS_SELECTOR, '[data-l10n-id="importCertificate"]')
     _select_certs_screen_locator = (By.ID, 'wifi-selectCertificateFile')
+    _select_active_wifi_locator = (By.CSS_SELECTOR, '.wifi-availableNetworks li.active')
```

Comment:
I think _active_wifi_locator is a better name, I don't know why the _select part is needed.